### PR TITLE
Add rails5.1

### PR DESCRIPTION
**Pull request by syguer**
Rails 5.1 has been released 🎉

### CODE REVIEW

File: lib/active_hash/base.rb [hunk 1]
```diff
@@ -35,7 +35,7 @@ def cache_key
         if Object.const_defined?(:ActiveModel)
           model_name.cache_key
         else
-          ActiveSupport::Inflector.tableize(self).downcase
+          ActiveSupport::Inflector.tableize(self.name).downcase
```
Comment (syguer) [inline lib/active_hash/base.rb]: I don't know why, but this code has been failed.

**Comment by syguer** [comment]
@zilkey @kbrock can you check?

**Comment by kbrock** [comment]
@syguer I currently have no projects that use active_hash.
I believe in the project and want to help where I can, but I have no corpus where I can run this.

Is there something in particular that I can do, or are you asking me to just kick the tires?
Keenan

**Comment by syguer** [comment]
@kbrock Thanks for reply. just kick the tires please!

**Comment by grosser** [comment]
need to get rid of quoted_id too see https://github.com/rails/rails/pull/27962#issuecomment-298498196
basically make the method not be defined if it's rails 5.1+

**Comment by syguer** [comment]
@grosser Thanks for information. It seems to be just deprecated. Could you create an issue about your problem?

**Comment by grosser** [comment]
here ya go: https://github.com/zilkey/active_hash/issues/150
should still be taken care of, deprecations are spammy :D

**Comment by syguer** [comment]
@grosser thanks! :)

**Pull request merged**